README.md

```diff
@@ -1,25 +1,19 @@
 <!-- Update the title -->
-# Terraform Modules Template Project
+# Terraform IBM Trusted Profile
 
-<!--
-Update status and "latest release" badges:
-  1. For the status options, see https://github.ibm.com/GoldenEye/documentation/blob/master/status.md
-  2. Update the "latest release" badge to point to the correct module's repo. Replace "module-template" in two places.
--->
 [![Incubating (Not yet consumable)](https://img.shields.io/badge/status-Incubating%20(Not%20yet%20consumable)-red)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
```

Comment:
This should probably have the stable badge as it will be stable once this Pr is merged